cloud_functions/climateiq_trigger_export_pipeline_cf/main.py

```diff
@@ -27,44 +36,45 @@ def trigger_export_pipeline(cloud_event: http.CloudEvent) -> None:
     Args:
         cloud_event: The CloudEvent representing the storage event.
     """
+    start = time.time()
+
     data = cloud_event.data
     object_name = data["name"]
 
+    # Structured logging:
+    # https://cloud.google.com/functions/docs/monitoring/logging#writing_structured_logs
+    print(json.dumps(dict(severity="DEBUG", message=f"[{object_name}] CF started")))
+
     # Extract components from the object name and determine the total number of
     # output prediction files.
-    #
-    # Note: Invalid object name is a non-retriable error so we return instead
-    # of throwing an exception (which would trigger retries).
     expected_format = (
         "<id>/<prediction_type>/<model_id>/<study_area_name>/"
         "<scenario_id>/prediction.results-<file_number>-of-{number_of_files_generated}"
     )
     path = pathlib.PurePosixPath(object_name)
     if len(path.parts) != 6:
-        print(
+        raise ValueError(
```

Comment:
are we planning on disabling retries?

Reply:
Yes, we're disabling retries. The I/O operations have their own retry mechanism.